pkg/executor/api.go

```diff
@@ -249,6 +249,62 @@ func (executor *Executor) unTapService(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 }
 
+// isValidHandler is very similar to getServiceForFunctionAPI, with the difference that the response returned doens't contain
+// the address, but instead returns "true" or "false", depending if the function service is valid or not.
+func (executor *Executor) isValidHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Sorry one more thing, you need to implement a method in `executor/client/client.go` which calls this endpoint (needs to be part of this PR because you are defining the request payload type).

Reply:
Added new method IsValid. Please take a look